sqlmesh/core/dialect.py

```diff
@@ -314,6 +317,13 @@ def _parse_join(
     return macro
 
 
+def _warn_unsupported(self: Parser) -> None:
+    self.__warn_unsupported()  # type: ignore
```

Comment:
i don't think we need to call super

Reply:
I kept it there to maintain the diagnostic message of which syntax is not supported; The following sqlmesh warning will not suffice for that matter. Wdyt, is it not needed anymore?

Reply:
you can make the current message more robust

Reply:
Not sure how to word this best for the average user who won't understand SQLGlot's `exp.Command`, SQLMesh's physical table references etc. Happy to change the message to whatever you prefer.